### ISSUE TITLE
Decorated pots are invisible

### ISSUE THREAD

**Issue by Gabrieljso** [FoundationGames/EnhancedBlockEntities#228]
decorated pots are invisible, when you right-click on them they appear but normally you can't see them.

**Comment by Makzevu** [comment]
Not sure if this is said somewhere else, but this happens when Sodium is installed with EnhancedBlockEntities. It took me until today to realize it. I fixed it on my end by installing Indium, but if that doesn't work for some reason, you can remove either mod as a workaround.

**Comment by Makzevu** [comment]
Also, not having Indium installed (or applying the workaround) makes chests double render to where opening a chest leaves a closed chest rendered under the open one.
![2024-06-28 15:35:05](https://github.com/FoundationGames/EnhancedBlockEntities/assets/86393728/938a7971-c856-4bd2-bcd4-93ec3b415091)

**Comment by Gabrieljso** [comment]
o

> Also, not having Indium installed (or applying the workaround) makes chests double render to where opening a chest leaves a closed chest rendered under the open one. ![2024-06-28 15:35:05](https://private-user-images.githubusercontent.com/86393728/344312230-938a7971-c856-4bd2-bcd4-93ec3b415091.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2MzA4NTYsIm5iZiI6MTcxOTYzMDU1NiwicGF0aCI6Ii84NjM5MzcyOC8zNDQzMTIyMzAtOTM4YTc5NzEtYzg1Ni00YmQyLWJjZDQtOTNlYzNiNDE1MDkxLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI5VDAzMDkxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM4YzdlYjU3MjE2YmE5YWRjYmNmODMwZDViZGY5NzFmNjRmYTcxNmExMjgzNmFlMzlkYjViYjk1NDFmZTAwN2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lqYR3LQqaVz5WvI8nwN7mzoa_xilKp-au4vcvNg0ZY0)

omg thank you

**Comment by Makzevu** [comment]
Yeah, I guess that EBE should check if Sodium is installed, and if so, require Indium.
